engine/crates/federation-audit-tests/tests/checkout.rs

```diff
@@ -0,0 +1,78 @@
+#![allow(unused_crate_dependencies, clippy::panic)]
+
+use std::process::Command;
+
+#[test]
+fn ensure_upstream_is_up_to_date() {
+    // The tests in this crate are powered by the upstream graphql-federation-gateway-audit repo
+    //
+    // I don't want people to have to manually fetch that and keep it up to date, because that sucks.
+    //
+    // But I also don't want to do the update inline in the audit_tests - because they'll probably
+    // be run in parallel and who can be bothered with cross process locking.
```

Comment:
> who can be bothered with cross process locking.

was going to suggest that xD
Doing that kind of thing with `OnceLock` / `tokio::sync::OnceCell` in clickhouse-integration-tests